python/run-tests

```diff
@@ -17,6 +17,65 @@
 # limitations under the License.
 #
 
+# Run test suites and indivisual test suite.
+#
+# Usage: run-tests [-v python version] [core|sql|mllib|ml|streaming]
+#
+# When you select none, all test suites are run. You can also select
+# multiple test suites.
+
+function usage() {
+    echo "Usage: run-tests [-v python version] [core|sql|mllib|ml|streaming]"
+}
+
+SUPPORT_PYTHON_VERSIONS="2.6 3.4 pypy"
```

Comment:
This is supposed to be an array I guess but this doesn't look like the typical Bash syntax for arrays?